source/tests/test_init_frz_model_se_atten.py

```diff
@@ -0,0 +1,158 @@
+import os, sys, platform, shutil, dpdata, json
+import numpy as np
+import unittest
+import subprocess as sp
+from common import j_loader, tests_path
+from deepmd.train.trainer import DPTrainer
+from deepmd.train.run_options import RunOptions
+from deepmd.utils.argcheck import normalize
+from deepmd.utils.compat import update_deepmd_input
+from deepmd.utils.data_system import DeepmdDataSystem
+
+from deepmd.env import GLOBAL_NP_FLOAT_PRECISION, tf
+from packaging.version import parse as parse_version
+
+if GLOBAL_NP_FLOAT_PRECISION == np.float32:
+    default_places = 4
+else:
+    default_places = 10
+
+
+def _file_delete(file):
+    if os.path.isdir(file):
+        os.rmdir(file)
+    elif os.path.isfile(file):
+        os.remove(file)
+
+
+def _subprocess_run(command):
+    popen = sp.Popen(command.split(), shell=False, stdout=sp.PIPE, stderr=sp.STDOUT)
+    for line in iter(popen.stdout.readline, b''):
+        if hasattr(line, 'decode'):
+            line = line.decode('utf-8')
+        line = line.rstrip()
+        print(line)
+    popen.wait()
+    return popen.returncode
+
+
+@unittest.skipIf(parse_version(tf.__version__) < parse_version("1.15"),
+    f"The current tf version {tf.__version__} is too low to run the new testing model.")
+class TestInitFrzModelA(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        data_file = str(tests_path / os.path.join("init_frz_model", "data"))
+        frozen_model = str(tests_path / "dp-frozen.pb")
+        ckpt_name = "init_frz_se_atten.ckpt"
+        ckpt = str(tests_path / ckpt_name)
+        run_opt_ckpt = RunOptions(init_model=ckpt, log_level=20)
+        run_opt_frz = RunOptions(init_frz_model=frozen_model, log_level=20)
+        INPUT = str(tests_path / "input.json")
+        jdata = j_loader(str(tests_path / os.path.join("init_frz_model", "input.json")))
+        jdata["training"]["training_data"]["systems"] = data_file
+        jdata["training"]["validation_data"]["systems"] = data_file
+        jdata["training"]["save_ckpt"] = ckpt_name
```

Comment:
```suggestion
        jdata["training"]["save_ckpt"] = ckpt
```

Reply:
Note that `ckpt_name` and `ckpt` is not the same thing when `tests_path` is not the current working directory.